spec-main/api-safe-storage-spec.ts

```diff
@@ -0,0 +1,83 @@
+import * as cp from 'child_process';
+import * as path from 'path';
+import { app, safeStorage } from 'electron/main';
+import { expect } from 'chai';
+import { emittedOnce } from './events-helpers';
+
+describe('safeStorage module', () => {
+  afterEach(() => {
+    app.quit();
+  });
+
+  describe('SafeStorage.isEncryptionAvailable()', () => {
+    it('should return true when encryption key is available', () => {
+      expect(safeStorage.isEncryptionAvailable()).to.equal(true);
+    });
```

Comment:
I don't suppose there's any way to test the `false` version of this?

Reply:
If I understand correctly we expect this to always be true? If @MarshallOfSound can confirm

Reply:
Yeah, we can force it to be false by locking the keychain I think, but then we can't unlock it in CI 😆 I think this is one of those APIs where we expect it to be true and if it isn't there's a problem 99% of the time